Dockerfile

```diff
@@ -27,5 +27,9 @@ USER ml-user
 COPY --chown=ml-user --from=builder /opt/quoting-service .
 RUN npm prune --production
 
+# Create empty log file & link stdout to the application log file
```

Comment:
This _may_ need to be added before the `RUN adduser` step to make sure that the correct permissions are applied to the log file, but I'm not 100% sure.

Reply:
Yeah, I figured that out when testing it. I just pushed an update to resolve it.

I also tested this on dev2, it seems to be all good now.